src/actions/index.js

```diff
@@ -1,14 +1,25 @@
+import { useDispatch } from 'react-redux';
 import axios from 'axios';
-
-import { GET_PAGES, MODAL_OPEN, MODAL_CLOSE } from './actionTypes';
+import { GET_PAGES, FETCH_CLUBS, MODAL_OPEN, MODAL_CLOSE } from './actionTypes';
 
 const api = 'https://dev.huoa.org/wp-json';
 
-export const getPages = () => async (dispatch) => {
-  const response = await axios.get(`${api}/wp/v2/pages?page=1&per_page=100`);
-  const data = response.data.reduce((allData, { slug, title, acf }) => ({ ...allData, [slug]: { title: title.rendered, ...acf }}), {});
+export const Initialize = () => { 
```

Comment:
I like how this is starting to look - although we should probably consider doing this somewhere else as its not the best practice to use a dispatch inside of our actions. Actions should ONLY be actions. Instead I say we create an `init.js` file in our `base` directory and do the dispatches there. Thoughts?

Reply:
sounds good to me working on it.

Reply:
done